app/models/work.rb

```diff
@@ -0,0 +1,23 @@
+class Work < ApplicationRecord
```

Comment:
I also suggest adding an `upvote` method to work, which takes a user and tries to add a vote to the current work.